Makefile

```diff
@@ -12,9 +12,24 @@ deps vendor_deps: check_setup
 	npm prune
 	echo "DONE.  Consider running:  git add -A vendor node_modules"
 
+# bump the version for upgrade
+version = v1.19.1
+# `go mod vendor` will reset vendor directory first time
+# so need to restore python and kolinter from vendor directory later
+go_deps go_vendor_deps: check_setup
+	@if [ ! -e ./go.mod ]; then go mod init khan_linter; fi
+	@if [ ! -e vendor/github.com/golangci/golangci-lint ]; then \
+		export GO111MODULE=on; \
+		go mod tidy; \
+		go mod vendor; \
+		git checkout vendor; \
+	fi
+	echo "DONE.  Consider running:  git add -A vendor"
 
 
 check_setup:
 	@command -v npm > /dev/null || echo "missing dependencies: need to install npm"
 	@command -v pip2 > /dev/null || echo "missing dependencies: need to install pip2"
 	@command -v pip3 > /dev/null || echo "missing dependencies: need to install pip3"
+	@if [ ! -e vendor/ktlint ]; then echo "missing dependencies: need to install ktlint"; fi
```

Comment:
Does this belong here?  It seems it's checking for package managers, not other linters.

Reply:
if just checking package managers, seems like here is not the right place to check linters.
I remove them from Makefile.